.github/workflows/pr.yml

```diff
@@ -12,9 +12,11 @@ jobs:
       - uses: actions/checkout@v3
 
       - name: Set Azurite Default Key
-        run: |
-          echo "AZURITE_ACCOUNT_KEY=${{ secrets.AZURITE_ACCOUNT_KEY }}" >> $GITHUB_ENV
-          echo "Using Azurite default key: $AZURITE_ACCOUNT_KEY"
+        run: echo "AZURITE_ACCOUNT_KEY=$(curl https://learn.microsoft.com/en-us/azure/storage/common/storage-use-azurite | grep "Account key:" | cut -b 24-111)" >> $GITHUB_ENV
```

Comment:
Is it possible to do this only when secrets is unset?

Reply:
Not concisely, because the act of referencing the secret is what causes it to fail on forks, from what I was seeing